### DISCUSSION TITLE
这是我的战争 求更新

### DISCUSSION THREAD

**Discussion by owl342655596** [Gamuxorg/bbs#256] [投递和请求游戏]
如题

**Comment by wangling12** [comment]
找了一圈，只能找到4.0.1的linux版本

**Comment by owl342655596** [comment]
magnet:?xt=urn:btih:963442077DF968EF113F93F004CFB5DC0308A8D7&tr=http%3A%2F%2Fbt.t-ru.org%2Fann%3Fmagnet

**Comment by owl342655596** [comment]
那个推荐的毛子的网站 6.0.7.3版本试试？

**Comment by owl342655596** [comment]
上面那个是mac的

**Comment by owl342655596** [comment]
好像最高版本就是4.0.1了Windows和mac还有Linux的版本号可能不同吧
